### ISSUE TITLE
Add bitcoin wallet

### ISSUE THREAD

**Issue by locrianz** [Pissandshittium/pissandshittium#44]
And maybe a way to use your bitcoins to give money to webdevs through the browser. And if they don't opt-in to receive the donations you can still donate but Pissandshittium Inc(TM)(R) will keep the bitcoins.

**Comment by EvilGremlin** [comment]
No, add wallet, bit with selection of bootom cryptocurrencies that cost less than nothing.
